composer.json

```diff
@@ -34,7 +34,8 @@
         "phpunit/phpunit": "^4.7|^5.0",
         "squizlabs/php_codesniffer": "^2.3",
         "jakub-onderka/php-parallel-lint": "^0.9.0",
-        "satooshi/php-coveralls": "^0.6.1"
+        "satooshi/php-coveralls": "^0.6.1",
+        "doctrine/orm": "^2.5"
```

Comment:
I believe so, `doctrine/dbal` can be used in isolation without `doctrine/orm`. If you remove this requirement the new test will fail as `doctrine/orm` isn't installed by Composer.